### ISSUE TITLE
Fix Responsive for SignupLoginPage.js and LoginPage.js

### ISSUE THREAD

**Issue by marleymarl** [marleymarl/geotime#94]
Only the right side Col (where the form is) should show up on Mobile.

**Comment by BaNazari** [comment]
Could be done, but what if the purpose of the empty left part? Will it show something in future? If there would be some data, it can be transferred to the section bellow the login section in mobile.

**Comment by marleymarl** [comment]
Yes you are right. In the left side will go marketing information e.g. bullet points to describe benefits of having account. It makes sense to have it appear below the form on mobile.

**Comment by BaNazari** [comment]
The color palette is different from website and the app. Are we gonna keep this one?

**Comment by marleymarl** [comment]
I'm not sure - it works for now. Once we add the marketing text to the left and also complete the website and dashboard, I think we can do a review of our styleguide and see how to bring it all together. Our style guide as it is was pretty rushed and we didn't do any kind of branding beforehand so we should do that work as a team and then come up with a new color scheme and brand guide according to that. In the meantime we can have some parts that don't match exactly.
